clients/src/main/java/org/apache/kafka/common/requests/DeleteTopicsResponse.java

```diff
@@ -53,8 +53,14 @@
      */
     private static final Schema DELETE_TOPICS_RESPONSE_V2 = DELETE_TOPICS_RESPONSE_V1;
 
+    /**
+     * v3 request is the same that as v2. The response is different based on the request version.
```

Comment:
Could add more specific description of the change? 
   "v3 response can return TOPIC_DELETION_DISABLED exception"

Reply:
Done!